src/main/java/org/junitpioneer/jupiter/Utils.java

```diff
@@ -12,13 +12,15 @@
 
 import java.lang.annotation.Annotation;
 import java.util.HashSet;
+import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collector;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.platform.commons.support.AnnotationSupport;
+import org.junit.platform.commons.util.AnnotationUtils;
```

Comment:
The `org.junit.platform.commons.util` package is not exported by module `org.junit.platform.commons`. It's **not** listed in the API documentation: https://junit.org/junit5/docs/current/api/ and `package-info.java` comments reads:

https://github.com/junit-team/junit5/blob/e80e2854c3862eef3feb08c2dd6276833ef911bd/junit-platform-commons/src/main/java/org/junit/platform/commons/util/package-info.java#L1-L9

Reply:
Good catch, thank you!
I wanted to use the method from AnnotationSupport but somehow one from AnnotationUtils was imported